feat(cwl): Allow cwl ctrl + click on whole line in an event stream

nkomonen-amazon opened the pull request:
## Problem
In cloudwatch logs when we search logs and ctrl + click we previously were only able to do this on the timestamp.

## Solution
Remove the limitation that only allows you to click on the timestamp and instead be able to click anywhere on the line.

Additionally, the whole line gets highlighted upon hover.

![Kapture 2023-01-30 at 18 58 19](https://user-images.githubusercontent.com/118216176/215623973-3d9e0905-8b4d-421d-b491-a7790e1d86c9.gif)


Signed-off-by: Nikolas Komonen <nkomonen@amazon.com>

<!---
    REMINDER:
    - Read CONTRIBUTING.md first.
    - Add test coverage for your changes.
    - Update the changelog using `npm run newChange`.
    - Link to related issues/commits.
    - Testing: how did you test your changes?
    - Screenshots
-->

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

justinmk3 commented:
Does this _highlight_ (underline) the whole line also? The main issue was that only small pieces of the timestamp were highlighted (underlined) on hover.

nkomonen-amazon commented:
> Does this _highlight_ (underline) the whole line also? The main issue was that only small pieces of the timestamp were highlighted (underlined) on hover.
@justinmk3 
Yup, I updated that after. Just updated the PR message.

justinmk3 commented:
Tested locally, this is an improvement. Remaining questions:

- holding ctrl (or cmd on macOS) and hovering the mouse over a line does a service call (which then triggers the cancelable dialog). Is there any way to prevent that?
- hover shows an empty box. if we don't have anything useful to show there, can we prevent this empty thing from showing? see screenshot

<img width="188" alt="Screen Shot 2023-01-30 at 4 03 22 PM" src="https://user-images.githubusercontent.com/55561878/215624615-012b4ce6-af3b-44eb-9d37-41c5431533c7.png">
